.github/workflows/issue_labeler.yml

```diff
@@ -0,0 +1,38 @@
+# Inspired by: https://github.com/stefanbuck/ristorante
+# See: https://stefanbuck.com/blog/codeless-contributions-with-github-issue-forms
+---
+
+name: Issue labeler
+on:
+  issues:
+    types: [ opened ]
+
+jobs:
+  label-component:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        template: [ bug_report.yml, feature_request.yml ]
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '16'
+
+      - name: Parse issue form
+        uses: stefanbuck/github-issue-praser@v2
+        id: issue-parser
+        with:
+          template-path: .github/ISSUE_TEMPLATE/${{ matrix.template }}
+
+      - name: Set labels based on component field
+        uses: redhat-plumbers-in-action/advanced-issue-labeler@v1
+        with:
+          issue-form: ${{ steps.issue-parser.outputs.jsonString }}
+          section: component
+          block-list: |
+            other
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I wonder why it needs this token? What kind of permissions does this action need?

Reply:
Token is used to set labels.

Source code: https://github.com/redhat-plumbers-in-action/advanced-issue-labeler/blob/b89eb39b97529bb7596cb4e716485a6d6f7dd74a/src/app.ts#L53-L55

Reply:
I think it should be set explicitly with https://github.blog/changelog/2021-04-20-github-actions-control-permissions-for-github_token/. All the other actions use it.

Reply:
Just to clarify as far as I understand this action accept random json on GitHub and then passes it to another action with 24 starts that hasn't been vetted by GitHub with the token. It seems risky to me but if it could only set labels it should probably be fine.

Reply:
Sure, I think it's a matter of preference, but I can update workflow to use permissions instead of token directly.

Reply:
> Just to clarify as far as I understand this action accept random json on GitHub and then passes it to another action with 24 stars that hasn't been vetted by GitHub with the token. It seems risky to me but if it could only set labels it should probably be fine.

Yes, this is correct. It can set only labels on issues. And you are one who defines labels in `advanced-issue-labeler.yml`

Reply:
> I think it should be set explicitly with https://github.blog/changelog/2021-04-20-github-actions-control-permissions-for-github_token/. All the other actions use it.

Ohh, I now get what you meant, I'll submit new PR.

Reply:
@evverx Please have a look at: #23811